e2e_tests/counterparty_test.py

```diff
@@ -146,4 +146,3 @@ def test_counterparty_dto():
     assert counterparty.attributes["permissions"] == counterparty_api_response["attributes"]["permissions"]
```

Comment:
we can also add tests for failures.. for example, update counterparty with verifyName = true that fails should return code - NameVerificationFailed